lib/distillery/plugins/link_config.ex

```diff
@@ -17,13 +17,17 @@ defmodule Releases.Plugin.LinkConfig do
   use Mix.Releases.Plugin
 
 
-  def before_assembly(_), do: nil
+  def before_assembly(_, _), do: nil
 
-  def after_assembly(_), do: nil
+  def after_assembly(_, _), do: nil
 
-  def before_package(_), do: nil
+  def before_package(_, _), do: nil
 
-  def after_package(%Release{version: version, profile: profile, name: name} = release) do
+  def after_package(_, _), do: nil
```

Comment:
@kanmo please group multiple clauses of a same function together  https://github.com/edeliver/edeliver/pull/274/files#diff-54433280d02e1346367d6df697291dd7R30

Reply:
I understand.